### ISSUE TITLE
evaluate stories260k on v2 dataset

### ISSUE THREAD

**Issue by jettjaniak** [delphi-suite/delphi#2]
Context: our test runs report much higher validation loss (on [v2 dataset](https://huggingface.co/datasets/jbrinkma/tinystories-v2-clean/viewer/default/validation)) than Karpathy's [models](https://github.com/karpathy/llama2.c?tab=readme-ov-file#models) (on v1 dataset). We don't know if the harder dataset explains that.

Task: Evaluate loss of smallest stories260k model trained by karpathy on v2 dataset's validation split. Make a PR with a notebook

**Comment by jannik-brinkmann** [comment]
stories260k evaluation on TinyStories v2 validation set gives loss of tensor(1.3859) [can add a notebook soon]; our best is 1.7 after 100k steps

**Comment by jannik-brinkmann** [comment]
The difference in validation loss could be traced back to the size of the tokenizer.
